curvefs/src/space/fast_align.h

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
src/client also has fast_align, Just quote it directly

Reply:
`fs` branch doesn't have this file. i will fix this later,